### DISCUSSION TITLE
Just here to say wouter is a godsend

### DISCUSSION THREAD

**Discussion by mkvlrn-cm42** [molefrog/wouter#398] [General]
I've dealt with react-router in the past, when it was version 4 or something. Then had to relearn all that crap for v5 for a single project. And now I just cannot and will not go over all that over engineering for v6 with remix or whatever the hel they are doing over there.

Thank you for this lib!

**Comment by molefrog** [comment]
Hello, thank you for such kind words, this really means a lot to me. I would say RR is a nicely designed library too and it is intended for slightly complex apps and use cases unlike wouter. But I def agree, i am a big fan of their classic API myself. Happy holidays.
